Dockerfile

```diff
@@ -9,8 +9,8 @@ VOLUME [ "/tmp", "/run/jetty" ]
 
 EXPOSE 8080
 
-COPY build/cas /var/lib/jetty/webapps/cas
-
+COPY --chmod=777 build/cas /var/lib/jetty/webapps/cas
```

Comment:
why such wide permissions ?

Reply:
I will check if 700 is enough

Reply:
@pmauduit  I have checked all possibility and app works only for 777, so I kept it like this. Thanks